### DISCUSSION TITLE
Json Error from i20230521 and rAudio 20230615

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1359] [Fixed Issues]
Steps to reproduce error

Create system from i20230521 (Raspberry Pi 3B+ Rev 1.3)
Open addons window
Update to rAudio 20230615
When complete press OK
Close addons window
Click on Systems window
JSON error immediately generated:  [JSON error.txt](https://github.com/rern/rAudio/files/11758088/JSON.error.txt)

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by Sagnasty** [comment]
#1354 is probably related as well

**Comment by Sagnasty** [comment]
Strange thing is that the 20230615 update is running fine (so far) on Raspberry Pi 3B Rev 1.2

I originally got the JSON error on my Raspberry Pi 3B+ Rev 1.3 after updating my from 20230609.   Hopefully the steps to reproduce from image will help resolve this quickly.

**Reply by Sagnasty** [reply]
Fixed in Audio 20230616 - Thank you for the quick resolution!

**Reply by Maikeleg** [reply]
Hi Stagnasty,
I'm the one who opened #1354 
Is there a possibility to do an update from commandline to 20230615 ?
Edit: 20230616 I mean

**Reply by Sagnasty** [reply]
Not sure - never had the need when the update function in the ADDON page already exists.

**Reply by ceejayemm** [reply]
@Maikeleg 
As @Sagnasty says just use the Addons / Updates functionality. Why make it harder ?

@Sagnasty - no problem/issues for me when updating RPi 3A, i20230621 / 20230615 to 20230616

Chris

**Reply by Maikeleg** [reply]
Thanks it works for me now, thanks for all work
